src/rating/rating.ts

```diff
@@ -17,12 +20,37 @@ export class NgbRating implements OnInit {
   private _oldRate: number;
   range: number[] = [];
 
+  /**
+   * Maximal rating that can be given using this widget.
+   */
   @Input() max: number = 10;
+
+  /**
+   * Current rating.
+   */
   @Input() rate: number;
-  @Input() readonly: boolean;
 
+  /**
+   * A flag indicating if rating could be read only (true) or updated (false).
```

Comment:
I find this a bit verbose. I don't think we need to explain what true or false means in this case. I mean, bit obvious from the name and the type of the input (boolean).